### ISSUE TITLE
cant bumpslash RC tank

### ISSUE THREAD

**Issue by Oscari-pog** [tgstation/TerraGov-Marine-Corps#11047]
### Description

The thing faster than a marine without armor must be clicked to be hit
This is hard when it is spazzing around and shredding you with the meme gun
Augh

### Test Merges

The following pull requests are currently test merged:
[#11011: 'Xenomorphs can pull objects again' by lKiseki at commit 1b3d6d408a](https://github.com/tgstation/TerraGov-Marine-Corps//pull/11011)
[#10998: 'Lets primordial hunters have stealth as well as disguise (again) but they cant be active at the same time' by iLikeMemes at commit 9ade2e8133](https://github.com/tgstation/TerraGov-Marine-Corps//pull/10998)
[#10610: 'Widow [MDB ignore]' by Zar000 at commit 82f535b701](https://github.com/tgstation/TerraGov-Marine-Corps//pull/10610)

### Reproduction Steps

1. Get attacked by RC Tank (A fast thing with gun)
2. Attempt to combat it by dancing
3. Haha no bumpslash fuck you
4. Die
5. Seethe

### Screenshots

![augh2](https://user-images.githubusercontent.com/70425197/192084113-b7bbbe13-bec9-47f8-992a-f587d5279ec4.png)

**Comment by ivanmixo** [comment]
This is rouny specific. Funny.

**Comment by isotope436** [comment]
mfw xenos cannot win without bump slash anymore like the good old days ;-;


now since I can I will explain UGV balance and why bumpslash shouldn't be added to the UGV not without balance changes to its health and/or armor. The UGV at current will die to a runner in about 3-6 hits (I don't have exacts) and that's because it actually has almost no health at all compared to its larger counterparts which are shit anyways because they have almost no speed (this is going off the engi UGV kit you can get at round start for 50 engi points that has the light chassis and the light gun) 

Now the easiest way to kill one of these thing is to NOT RUN FROM IT [GOD I CANNOT STRESS THIS ENOUGH IF YALL ACTUALLY TRIED FIGHTING THE FUCKING THING YOU WOULD REALIZE HOW EASY IT IS TO SCARE OFF] . Xeno fear factor is the ONLY reason I get 90% of my kills, yall cant see its hp and just assume its a super speed demi god and just DONT FIGHT IT 

once you guys realize the longer its on your ass the more damage its gonna do you will start realizing the easiest counter to it is just throwing hands. The little gun it has may have 150 rounds but they are just renamed SMG rounds they don't do that much damage (unlike the heavy UGV gun) and take a SHIT ton of time to actually put rounds on target (at least enough to matter)

If xenos got bumpslash on it the basic UGV (light gun and chassis a combo that costs almost 100-120 req points) would basically die to any and all runners it encounters. runners at current are about twice as fast as the UGV is and with their 100% evade skill its basically gg ez for a UGV so long as the runner can bumpslash well, especially when you consider the UGV guns are HARD LOCKED to semi auto and really buggy with the delay (sometimes you can blast the fuck out of it and sometimes it just wont read any inputs at all) 

not only that but the UGV's already actually have hard counters in the game. boiler, ganking, entrapment (as it cannot go past a xeno), and acid (especially direct hit boiler balls). These are all very good counters to it but the best and most common HARD KILL counter is landing a crusher charge on it as this will vaporize it INSTANTLY (90% of my deaths in the UGV are to crusher charges)

basically because its rare and xenos have almost NO experience against it they don't realize or know how to kill it.

tldr: xeno skill issue

**Comment by Oscari-pog** [comment]
/\ me when I make up things so my favorite thing about UV doesnt get removed

**Comment by Oscari-pog** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/issues/11211#issue-1405560643

This is purely a rouny issue which Elsa correctly described so ill be closing this
